README.md

```diff
@@ -62,13 +62,16 @@ This is the easiest way to get started for development with the permissioning Da
 ### Deploying the contracts
 1. The [Besu documentation](https://besu.hyperledger.org/en/stable/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning/)
    describes how to use the contracts for onchain permissioning with Besu, including setting environment variables.
-
-2. The following additional environment variables are optional and can be used to permit accounts and nodes during initial contract deployment
+2. The following additional environment variables are optional and can be used to prevent redeployment of rules contracts. If set to true, that contract will not be redeployed and current list data will be preserved.
```

Comment:
Should we mention the default behaviour?

Reply:
done